src/syscall_table/x86_64.rs

```diff
@@ -1,17 +1,17 @@
-// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
 
 // This file is auto-generated by `tools/generate_syscall_tables`.
 // Do NOT manually edit!
-// Generated on: Mon Jan 17 17:30:54 UTC 2022
-// Kernel version: 5.10
+// Generated on: Sat Dec 14 01:47:02 PM CST 2024
+// Kernel version: 6.12
 
 use std::collections::HashMap;
 
 pub(crate) fn make_syscall_table() -> HashMap<&'static str, i64> {
     vec![
-        ("accept", 43),
         ("accept4", 288),
+        ("accept", 43),
```

Comment:
looks like the `sort` you used when building this is different than the one used in the rust vmm container. you could use that container image to run the script